test/uk/gov/hmrc/tai/repositories/EmploymentRepositorySpec.scala

```diff
@@ -1161,7 +1176,7 @@ class EmploymentRepositorySpec extends BaseSpec {
       val annualAccountCY = createAnnualAccount(taxYear = currentTaxYear)
       val annualAccountPY = createAnnualAccount(taxYear = previousTaxYear)
 
-      val employment1Id = 4
+      val employment1Id = 1
```

Comment:
This should still be 4?

Reply:
val employment1Id is being used as sequence number in this test and to match this employment with Annual Account(which has sequence number as 1) I changed it to 1.